docs/getting_oss_rules_of_play_rating.md

```diff
@@ -23,6 +23,10 @@ The data provider `CodeOfConductGuidelineInfo` allows you to specify 1 type of r
 - The content that you expect to be part of the code of conduct guidelines.
 The option is called `requiredContentPatterns` and expects a list of one Regex. If you enter multiple entries, the code of conduct guidelines must comply with all the patterns for the rule to pass.
 
+### Code Scan Alerts
+The data provider `CodeqlDataProvider` allows you to check if the code scans are enabled for the project or not.
+It also checks if the code scans are availble on pull requests or not.
```

Comment:
We we are checking for a specific code scan `CodeQL` please be specific. A reference URL is always good. Please rephrase this text about the use case a bit better? Maybe with bullet points as there are 2 things that the CodeqlDataProvider collects.